[2025-02-23T16:00:04.289+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-23T16:00:04.334+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: london_weather_pipeline.transform_weather_data scheduled__2025-02-23T15:00:00+00:00 [queued]>
[2025-02-23T16:00:04.360+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: london_weather_pipeline.transform_weather_data scheduled__2025-02-23T15:00:00+00:00 [queued]>
[2025-02-23T16:00:04.361+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-23T16:00:04.395+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): transform_weather_data> on 2025-02-23 15:00:00+00:00
[2025-02-23T16:00:04.402+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=21066) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-23T16:00:04.405+0000] {standard_task_runner.py:72} INFO - Started process 21071 to run task
[2025-02-23T16:00:04.406+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'london_weather_pipeline', 'transform_weather_data', 'scheduled__2025-02-23T15:00:00+00:00', '--job-id', '152', '--raw', '--subdir', 'DAGS_FOLDER/london_weather_pipeline/london_weather_etl.py', '--cfg-path', '/tmp/tmpwpoqh9dp']
[2025-02-23T16:00:04.407+0000] {standard_task_runner.py:105} INFO - Job 152: Subtask transform_weather_data
[2025-02-23T16:00:04.500+0000] {task_command.py:467} INFO - Running <TaskInstance: london_weather_pipeline.transform_weather_data scheduled__2025-02-23T15:00:00+00:00 [running]> on host ee2081ae3a04
[2025-02-23T16:00:04.684+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='london_weather_pipeline' AIRFLOW_CTX_TASK_ID='transform_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-23T15:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-23T15:00:00+00:00'
[2025-02-23T16:00:04.685+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-23T16:00:04.689+0000] {python.py:240} INFO - Done. Returned value was: {'latitude': '51.5074', 'longitude': '-0.1278', 'temperature': 11.4, 'windspeed': 23.2, 'winddirection': 204, 'weathercode': 3}
[2025-02-23T16:00:04.738+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-23T16:00:04.738+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=london_weather_pipeline, task_id=transform_weather_data, run_id=scheduled__2025-02-23T15:00:00+00:00, execution_date=20250223T150000, start_date=20250223T160004, end_date=20250223T160004
[2025-02-23T16:00:04.821+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-23T16:00:04.862+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-23T16:00:04.865+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
